samples/msal-browser-samples/HybridSample/README.md

```diff
@@ -0,0 +1,198 @@
+# Hybrid SPA sample application
```

Comment:
We should also link to this sample from the browser and node READMEs and FAQs

Reply:
Good call.